data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2194,10 +2278,38 @@
             x,y=36,20
         [/filter]
 
-        [message]
-            speaker=unit
-            message= _ "Ugh, a corpse. And it — (<i>gags</i>) — reeks!"
-        [/message]
+        [if]
+            [variable]
+                name=unit.id
+                equals="Abhai"
+            [/variable]
+            [then]
+                [message]
+                    speaker=Abhai
+                    message= _ "This corpse has been here for some time. I am glad I can no longer smell."
+                [/message]
+            [/then]
+            [else]
+                [if]
+                    [variable]
+                        name=unit.id
+                        equals="Krash"
+                    [/variable]
+                    [then]
+                        [message]
+                            speaker="Krash"
+                            message= _ "Body. Stinks."
```

Comment:
While this line is understandable when reading the WML, where we know what triggered it and what the normal unit's line is, it doesn't seem so understandable out of context. If a player is moving a group of units up this river, a sudden "Body. Stinks." is a question of what body? Has it triggered because of the hex, or is Krash saying that his own body is decomposing - should the player prepare for the drake to turn into an enemy undead?

Reply:
Seeing this as "Krash's way of speaking" makes me think that someone went a _little_ overboard at making him a man of few words. I think you could add up to 7 more words and still qualify as "choppy, blunt, and few words". Not that you'd need to for this particular case – 3-4 words is probably perfectly sufficient to get across a meaning roughly equivalent to the more verbose one.

I think at least adding the word "here" somewhere should help.

Reply:
This is an outdated review thread, the current text now reads 'Corpse... Stinks.' as per the above response.

In response to CelMin: 'Body here. It stinks.' - Better?